src/lib/finishPollMessage.ts

```diff
@@ -25,6 +25,15 @@ async function finishPoll(poll: IPoll, { persis }: { persis: IPersistence }) {
     return persis.updateByAssociation(association, poll);
 }
 
+async function getuserTimezone(read: IRead, data) {
```

Comment:
Missing type for `data`

Reply:
Because the type of data was  `any`
 But i can give it type in this function according to usage.